app/api/helpers/storage.py

```diff
@@ -162,7 +162,7 @@ def upload_local(uploaded_file, key, **kwargs):
         return get_settings()['static_domain'] + \
             file_relative_path.replace('/static', '')
     url = urlparse(request.url)
-    return url.scheme + '://' + url.hostname + file_relative_path
+    return '%s://%s:%s%s' % (url.scheme, url.hostname, url.port, file_relative_path)
```

Comment:
Use '{scheme}://{hostname}:{port}{file_relative_path}'.format(scheme=url.scheme, hostname=url.hostname, port=url.port, file_relative_path=file_relative_path)

Reply:
@rafalkowalski Thanks, I'll update it.